src/coreclr/vm/method.hpp

```diff
@@ -162,8 +162,7 @@ enum MethodDescFlags
 struct MethodDescCodeData final
 {
     PTR_MethodDescVersioningState VersioningState;
-
-    // [TODO] Move temporary entry points here.
+    PCODE m_pTemporaryEntryPoint;
```

Comment:
```suggestion
    PCODE TemporaryEntryPoint;
```

This is a public field and I'd prefer to avoid this sort of hungarian notion.